droid-binary/assembly.xml

```diff
@@ -98,23 +98,11 @@
       <fileMode>644</fileMode>
     </file>
     <file>
-      <source>bin/Running DROID.txt</source>
+      <source>bin/sigtool.bat</source>
       <outputDirectory>/</outputDirectory>
       <filtered>true</filtered>
       <fileMode>644</fileMode>
     </file>
-    <file>
-      <source>bin/droid.sh</source>
```

Comment:
This looks like it removes the sigtool and droid.sh scripts from the multi-platform package. Is that intentional? These files are needed for non-Windows users.